packages/esm-patient-chart-app/src/deceased/deceased.resource.ts

```diff
@@ -0,0 +1,35 @@
+import { openmrsFetch, openmrsObservableFetch } from '@openmrs/esm-framework';
+
+export async function setDeceased(payload: any, personUuid: string, abortController: AbortController) {
+  return openmrsFetch(`/ws/rest/v1/person/${personUuid}`, {
```

Comment:
Do we have an `openmrsPost` ? It would be nice if we have a project wide concept of posting including its artifacts {post, submissinInProgress, result, error, (reset)}.

Reply:
I don't think we do have an `openmrsPost`, we usually use `openmrsFetch`